swagger.yaml

```diff
@@ -47,6 +47,32 @@ paths:
             type: string
           description: Server error
 
+  /createChatRoom:
```

Comment:
It should be 'create-chat-room,' which follows the basic convention used in APIs

Reply:
changed